Controller/TutorialController.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace SumoCoders\FrameworkExampleBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Validator\Constraints\DateTime;
+
+class TutorialController extends Controller
+{
+    /**
+     * @Route("/tutorial/datepicker")
+     * @Template()
+     * @param Request $request
+     * @return array
+     */
+    public function datePickerAction(Request $request)
+    {
+        $defaultData = array(
```

Comment:
Die default data zou ik eigenlijk ook in de FormType steken. Ge kunt da voor ieder field definen, en in zo'n voorbeeld lijkt het me logischer dat de data en het form field zo dicht mogelijk bij elkaar liggen.